### PR TITLE
Correct OF nodelet type name so the system monitor can load it

### PR DESCRIPTION
**Pull request by rgarciaruiz**

### CODE REVIEW

**Comment by rgarciaruiz** [comment]
One of the tests fail because #811 fixes that. Should we merge anyway or wait until #811 is merged to develop?

**Review by kbrowne15 — APPROVED** [review]

**Comment by kbrowne15** [comment]
It's up to you. I did approve #811 so if you want to merge that in first, you can. Otherwise, the failed test doesn't relate to this pr so I'm okay with you merging it in even with a failing test.

**Pull request merged**
